CHANGELOG.md

```diff
@@ -163,3 +163,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [2.0.1] - 2022-06-02
 - Last released version before CHANGELOG existed.
 
+### Fixed
```

Comment:
@isc-akadoo this belongs under "Fixed" the 2.5.0 section for the upcoming release.

Reply:
Moved to the upcoming release section.